logging/bin/getlogs.py

```diff
@@ -92,7 +107,21 @@ def validate_input(dict):
         if ((not ".json" in dict['savequery'])):
             print('Error: Not a supported filetype for the saved query file.')
             exit()
+        if (not "GetLogs_Files/" in dict['savequery']):
+            dict['savequery']= os.path.join("GetLogs_Files", dict['savequery'])
 
+    if dict['query-filename']:
+        if (not "GetLogs_Files/" in dict['query-filename']):
```

Comment:
Again, future update, but since we keep doing this check for directory name and update if the directory isn't present maybe we should turn this into a function.